sql/moz-fx-data-shared-prod/firefox_desktop/newtab/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# yamllint disable rule:line-length
+friendly_name: Pings for `firefox-desktop/newtab`
+description: |-
+  A view of pings sent for the
+  `firefox-desktop/newtab`
+  document type.
+
+  Clustering fields: `submission_timestamp`
+workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:pocket/prefect
```

Comment:
There's currently a "bug" where table-level metadata should be redundant with dataset-level data but isn't, so you should add `workgroup:mozilla-confidential` explicitly to this list and the list in `newtab_live` (which should be undone later when we fix that bug). I will file a bqetl issue for this if one doesn't exist.